lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -212,8 +212,7 @@ def safe_bulk(actions)
         retry unless @stopping.true?
       rescue ::LogStash::Outputs::ElasticSearch::HttpClient::Pool::BadResponseCodeError => e
         if RETRYABLE_CODES.include?(e.response_code)
-          safe_url = ::LogStash::Outputs::ElasticSearch::SafeURL.without_credentials(e.url)
-          log_hash = {:code => e.response_code, :url => safe_url}
+          log_hash = {:code => e.response_code, :url => e.url}
```

Comment:
can we maybe log the url.to_s or similar? otherwise unecessary object ids leak to the logs (visual garbage):

```
[2016-12-15T12:06:43,688][ERROR][logstash.outputs.elasticsearch] Attempted to send a bulk request to elasticsearch but received a bad HTTP response code! {:code=>503, :url=>#<URI::HTTP:0x4189e152 URL:http://127.0.0.1:9202/_bulk>}
```